### PR TITLE
e2e: update podpools tests to pass with new cpuallocator

### PR DESCRIPTION
**Pull request by askervin**
Update podpools tests and make them easy to update when CPU allocator behavior changes next time.

This PR runs on top of PR #602.

### CODE REVIEW

**Comment by klihub** [comment]
This LGTM, once @marquiz's PR is merged.

**Review by klihub — APPROVED** [review]
LGTM.

**Review by marquiz — DISMISSED** [review]
LGTM

**Review by kad — APPROVED** [review]

**Review by klihub — APPROVED** [review]

**Pull request merged**
